src/AdminConsole/appsettings.Development.json

```diff
@@ -7,6 +7,8 @@
     }
   },
   "Stripe": {
+    "ApiKey": "sk_test_51N0oi4BFpfsE1ibiseLnBVFC9AZdx0cy2VjZPJBGCDLzcjvIjOmEsZ2DKIiuBg92kVriYjxKJVZJDfbBj43KEMrT00lfYS5Y2R",
```

Comment:
@jonashendrickx We do not want to put this into source control. I will regenerate this data to make sure our testing data is not publicly accessible.